src/js/state/helpers.js

```diff
@@ -463,8 +463,7 @@ export function createInitialState(formConfig) {
     reviewPageView: {
       openChapters: [],
       viewedPages: new Set()
-    },
-    trackingPrefix: formConfig.trackingPrefix
```

Comment:
We don't need `trackingPrefix` anymore since that data would be held by the app and depend on what service they used.

Reply:
I'm confused how this part would work. It seems like they may still want a way to differentiate between events of the same name that happen on different forms, so it seems like this would still be needed in our code when we log events?